src/contexts/BalancesProvider.tsx

```diff
@@ -236,11 +234,11 @@ export function BalancesProvider({ children }: { children: any }) {
       return;
     }
 
-    if (isPolling) {
-      const tokenTypes = Object.entries(subscribers)
-        .filter(([, subscriberCount]) => subscriberCount > 0)
-        .map(([tokenType]) => tokenType as TokenType);
+    const tokenTypes = Object.entries(subscribers)
```

Comment:
This fixes a bug which was causing us to still actively poll balances on screens which don't necessarily need them.